internal/manifest/manifest.go

```diff
@@ -154,6 +159,16 @@ func UpdateSources(manifest models.ApplicationManifest, cmd *cobra.Command) (mod
 		kind = models.OriginGit
 		origins++
 
+		// Standard provider, and conditional override by the user
+		gitRef.Provider = models.ProviderGit
```

Comment:
I was thinking that maybe we could also check for the standard urls for Github and Gitlab:

```go
func GitProviderFromOriginURL(url string) {
	u, err := url.Parse("https://github.com/ndcsjk/sdhbnis")
	// check for u.Host == github.com or gitlab.com
}
```
and then 
```go
provider, err := models.GitProviderFromOriginURL(gitURL)
if err != nil {
	// log something about the error
	gitRef.Provider = models.ProviderGit
} else {
	gitRef.Provider = provider
}
```

What do you think? The `--git-provider` could be provided anyway to overwrite this behaviour.

Reply:
I feel the beginning of feature creep here. Willing to try this one thing. And CI just went :heavy_check_mark: :(

Reply:
See commit 14309cf6159097d6d95aff0be97dfe35a0bdc498